### PR TITLE
Don't check for debugDidSendFirstFrameEvent when adding service extensions for Dart VM apps.

### PR DESCRIPTION
**Pull request by kenzieschmoll**
eval_on_dart_library was throwing an error because flutter libs are not available for DartVM apps. For dart vm apps, we should add service extensions immediately.

### CODE REVIEW

File: packages/devtools_app/lib/src/service_manager.dart [hunk 1]
```diff
@@ -513,42 +517,48 @@ class ServiceExtensionManager {
...
-          );
+        if (!_firstFrameEventReceived) {
+          bool didSendFirstFrameEvent = false;
+          if (isServiceExtensionAvailable(extensions.didSendFirstFrameEvent)) {
+            final value = await _service.callServiceExtension(
+              extensions.didSendFirstFrameEvent,
+              isolateId: _isolateManager.selectedIsolate.id,
+            );
+            didSendFirstFrameEvent =
+                value != null && value.json['enabled'] == 'true';
```
Comment (bkonyi) [inline packages/devtools_app/lib/src/service_manager.dart]: Could be:

```dart
didSendFirstFrameEvent = value?.json['enabled'] == true;
```

File: packages/devtools_app/lib/src/service_manager.dart [hunk 2]
```diff
@@ -513,42 +517,48 @@ class ServiceExtensionManager {
...
+          } else {
+            final EvalOnDartLibrary flutterLibrary = EvalOnDartLibrary(
+              [
+                'package:flutter/src/widgets/binding.dart',
+                'package:flutter_web/src/widgets/binding.dart',
```
Comment (bkonyi) [inline packages/devtools_app/lib/src/service_manager.dart]: Is this still necessary? We've deprecated package:flutter_web now that it's been merged into the main framework.

File: packages/devtools_app/lib/src/service_manager.dart [hunk 3]
```diff
@@ -513,42 +517,48 @@ class ServiceExtensionManager {
...
+          }
         }
-
-        if (didSendFirstFrameEvent) {
-          await _onFrameEventReceived();
+      } else {
+        for (String extension in isolate.extensionRPCs) {
+          await _addServiceExtension(extension);
         }
       }
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/service_manager.dart]: @jacob314 we are adding service extensions for non flutter apps here

File: packages/devtools_app/lib/src/service_manager.dart (hunk 1 above)
Comment (kenzieschmoll) [reply packages/devtools_app/lib/src/service_manager.dart]: done

File: packages/devtools_app/lib/src/service_manager.dart (hunk 2 above)
Comment (kenzieschmoll) [reply packages/devtools_app/lib/src/service_manager.dart]: removed

**Pull request merged**